### DISCUSSION TITLE
Pending backport rules make it unclear that the original PR is ready to go

### DISCUSSION THREAD

**Discussion by HoloRin** [Mergifyio/mergify#3174] [Workflows]
We have some automatic backporting triggered based or PR labels. It works great, but, the pending backport rule makes it look as if the PR is not yet ready to be merged, even though it is. I was wondering if there was a way to address this. See below for an example of the Github UI in this state:
<img width="951" alt="Screen Shot 2021-09-14 at 15 15 30" src="https://user-images.githubusercontent.com/217064/133265688-11c53fd9-90a3-4559-8c12-d00795d3613a.png">

**Comment by jd** [comment]
That's probably something we should change indeed. Thoughts @sileht?

**Comment by sileht** [comment]
Yes, this is something we plan to change, the check-run will not be post until the pull request is merged.

**Reply by HoloRin** [reply]
Glad to hear that, I think it will be a nice improvement.

**Reply by jd** [reply]
The fix is https://github.com/Mergifyio/mergify-engine/pull/3186
